sdk/eventgrid/Azure.Messaging.EventGrid/tests/ConsumeEventTests.cs

```diff
@@ -1343,55 +1343,56 @@ public void ParsesCloudEventEnvelope()
-        }
+        //    events[0].ExtensionAttributes.TryGetValue("key", out var value);
+        //    Assert.AreEqual("value", value);
+        //}
 
-        [Test]
-        public void ConsumeCloudEventUsingBinaryDataExtensionMethod()
-        {
-            BinaryData messageBody = new BinaryData("{  \"source\": \"/subscriptions/id/resourceGroups/Storage/providers/Microsoft.Storage/storageAccounts/xstoretestaccount\",  \"subject\": \"/blobServices/default/containers/testcontainer/blobs/testfile.txt\",  \"type\": \"Microsoft.Storage.BlobDeleted\",  \"time\": \"2017-11-07T20:09:22.5674003Z\",  \"id\": \"4c2359fe-001e-00ba-0e04-58586806d298\",  \"data\": {    \"api\": \"DeleteBlob\",    \"requestId\": \"4c2359fe-001e-00ba-0e04-585868000000\",    \"contentType\": \"text/plain\",    \"blobType\": \"BlockBlob\",    \"url\": \"https://example.blob.core.windows.net/testcontainer/testfile.txt\",    \"sequencer\": \"0000000000000281000000000002F5CA\",   \"brandNewProperty\": \"0000000000000281000000000002F5CA\", \"storageDiagnostics\": {      \"batchId\": \"b68529f3-68cd-4744-baa4-3c0498ec19f0\"    }  }}");
+        //[Test]
+        //public void ConsumeCloudEventUsingBinaryDataExtensionMethod()
+        //{
+        //    BinaryData messageBody = new BinaryData("{  \"source\": \"/subscriptions/id/resourceGroups/Storage/providers/Microsoft.Storage/storageAccounts/xstoretestaccount\",  \"subject\": \"/blobServices/default/containers/testcontainer/blobs/testfile.txt\",  \"type\": \"Microsoft.Storage.BlobDeleted\",  \"time\": \"2017-11-07T20:09:22.5674003Z\",  \"id\": \"4c2359fe-001e-00ba-0e04-58586806d298\",  \"data\": {    \"api\": \"DeleteBlob\",    \"requestId\": \"4c2359fe-001e-00ba-0e04-585868000000\",    \"contentType\": \"text/plain\",    \"blobType\": \"BlockBlob\",    \"url\": \"https://example.blob.core.windows.net/testcontainer/testfile.txt\",    \"sequencer\": \"0000000000000281000000000002F5CA\",   \"brandNewProperty\": \"0000000000000281000000000002F5CA\", \"storageDiagnostics\": {      \"batchId\": \"b68529f3-68cd-4744-baa4-3c0498ec19f0\"    }  }}");
 
-            CloudEvent cloudEvent = messageBody.ToCloudEvent();
+        //    CloudEvent cloudEvent = messageBody.ToCloudEvent();
 
-            Assert.NotNull(cloudEvent);
-            switch (cloudEvent.Type)
-            {
-                case SystemEventNames.StorageBlobDeleted:
-                    StorageBlobDeletedEventData blobDeleted =cloudEvent.GetData<StorageBlobDeletedEventData>();
-                    Assert.AreEqual("https://example.blob.core.windows.net/testcontainer/testfile.txt", blobDeleted.Url);
-                    break;
-            }
-        }
+        //    Assert.NotNull(cloudEvent);
+        //    switch (cloudEvent.Type)
+        //    {
+        //        case SystemEventNames.StorageBlobDeleted:
+        //            StorageBlobDeletedEventData blobDeleted =cloudEvent.EventData.ToObjectFromJson<StorageBlobDeletedEventData>();
```

Comment:
?

Reply:
There is a TODO to add these back once the autorest issue about adding converters is fixed. I'll create an issue.